server/socketio-server.js

```diff
@@ -76,7 +76,7 @@ io.on("connection", socket => {
       const occupants = rooms[curRoom].occupants;
       socket.to(curRoom).broadcast.emit("occupantsChanged", { occupants });
 
-      if (occupants == {}) {
+      if (JSON.stringify(occupants) === JSON.stringify({})) {
```

Comment:
You're right
```
occupants={}
occupants == {}
false
```

Instead of jsonify to do a comparison.
`if (Object.keys(occupants).length === 0)`
should be better, this is what open-easyrtc more or less do via the underscore size function.
Can you please test that and tell me if it's good for you?

Reply:
`if (Object.keys(occupants).length === 0)
`
works as well